app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java

```diff
@@ -360,13 +360,16 @@ private void checkTrojanURLFromClipboard() {
         Threads.instance().runOnUiThread(new Runnable() {
             @Override
             public void run() {
-                ClipboardManager mClipboardManager = (ClipboardManager) getSystemService(CLIPBOARD_SERVICE);
+                ClipboardManager clipboardManager = (ClipboardManager) getSystemService(CLIPBOARD_SERVICE);
                 // check clipboard
-                if (!mClipboardManager.hasPrimaryClip() || mClipboardManager.getPrimaryClip().getItemCount() == 0) {
+                if (!clipboardManager.hasPrimaryClip() || clipboardManager.getPrimaryClip().getItemCount() == 0) {
                     return;
                 }
 
-                final CharSequence clipboardText = mClipboardManager.getPrimaryClip().getItemAt(0).getText();
+                final CharSequence clipboardText = clipboardManager.getPrimaryClip().getItemAt(0).getText();
```

Comment:
I think we should use https://developer.android.com/reference/android/content/ClipData.Item#coerceToText(android.content.Context)

Reply:
> I think we should use https://developer.android.com/reference/android/content/ClipData.Item#coerceToText(android.content.Context)

Reasonable, fixed already.